examples/apps/data-object-grid/tsconfig.json

```
@@ -1,10 +1,9 @@
 {
-	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"extends": "../../../common/build/build-common/tsconfig.base.json",
```

I suggest still extending the node16 config here.

```suggestion
	"extends": "../../../common/build/build-common/tsconfig.node16.json",
```

---

Why? (I just made an independent comment that we should not extend from Node16 config when we aren't using anything from it.)